javascript-loops/main.js

```diff
@@ -0,0 +1,68 @@
+/* exported getNumbersToTen,
+            getEvenNumbersToTwenty,
+            repeatWord,
+            logEachCharacter,
+            doubleAll,
+            getKeys,
+            getValues
+ */
+function getNumbersToTen() {
+  var number = [];
+  var currentNumber = (0);
+  while (currentNumber < 10) {
+    currentNumber++;
+    number.push(currentNumber);
+  }
+  return number;
+}
```

Comment:
Counting should start from `1`:

---

![image](https://user-images.githubusercontent.com/16143771/145317800-e107b318-3b98-4ba8-abe2-cd16b927b12d.png)